src/Microsoft.Extensions.Configuration/ConfigurationBuilderExtensions.cs

```diff
@@ -7,9 +7,42 @@
 
 namespace Microsoft.Extensions.Configuration
 {
-    public static class MemoryConfigurationExtensions
+    public static class ConfigurationBuilderExtensions
```

Comment:
Since there were two different providers, I didn't have a great name, DefaultConfigurationBuilderExtensions is the latest stab... feel free to propose something less lame :)